components/organisms/o-personal-details.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+  <div class="o-personal-details">
+    <SfHeading
+      :title="`1. ${$t('Personal Details')}`"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <div class="form">
+      <SfInput
+        v-model.trim="personalDetails.firstName"
+        class="form__element form__element--half"
+        name="first-name"
+        :label="$t('First name *')"
+        :valid="!$v.personalDetails.firstName.$error"
+        :error-message="
+          !$v.personalDetails.firstName.required
+            ? $t('Field is required')
+            : !$v.personalDetails.firstName.minLength
+              ? $t('Name must have at least 2 letters.')
+              : ''
+        "
+        @blur="$v.personalDetails.firstName.$touch()"
+      />
+      <SfInput
+        v-model.trim="personalDetails.lastName"
+        class="form__element form__element--half form__element--half-even"
+        name="last-name"
+        :label="$t('Last name *')"
+        :valid="!$v.personalDetails.lastName.$error"
+        :error-message="
```

Comment:
Error message can be `:error-message="$t('Field is required')"` (without condition)

Reply:
Done.